spec/models/run_spec.rb

```diff
@@ -180,14 +180,10 @@
       expect(run.total_playtime_ms(Run::REAL)).to eq 77_066_934
     end
 
-    it 'reports no history using fast parsing' do
-      expect(run.parse(fast: true)[:realtime_history]).to be_nil
-    end
-
     it 'reports correct history when using slow parsing' do
```

Comment:
"when using slow parsing" can be chopped off of this sentence it looks like

Reply:
How does it look now?  Tried to make it sound a bit better.